javaspec-runner/src/test/resources/info/javaspecfeature/TestRunner.feature

```diff
@@ -22,3 +22,16 @@ Feature: Running tests
     Given I have a class with JavaSpec tests in it that is marked to run with a JavaSpec runner
     When I run the tests with a JUnit runner
     Then the test runner should run all the tests in the marked class
+
+  Scenario: Failing tests
```

Comment:
Acceptance tests that would have helped me identify the root cause, had they been present earlier.